README.md

```diff
@@ -119,6 +119,45 @@ After running `docker-compose up` command, you can run any application you want.
     }
 </details>
 
+<details>
+    <summary>With Non-Transactional Retry</summary>
+
+    func main() {
+        consumerCfg := &kafka.ConsumerConfig{
+            Reader: kafka.ReaderConfig{
+                Brokers: []string{"localhost:29092"},
+                Topic:   "standart-topic",
+                GroupID: "standart-cg",
+            },
+            LogLevel:     kafka.LogLevelDebug,
+            RetryEnabled: true,
+            NonTransactionalBatchRetryEnabled: true,
```

Comment:
Bunu `BatchConfiguration` içerisine mi alsak? `NonTransactionalFailureStrategy` mi desek acaba 👀

Reply:
Don't use double negatives on variables :D We could add TransactionalRetry as a default behavior and enable non-transactional when it is false.